crates/codegen/runtime/cargo/src/runtime/cli/commands/bindings.rs

```diff
@@ -3,10 +3,12 @@ use core::fmt;
 use semver::Version;
 
 use super::CommandError;
-use crate::bindings::{self, Bindings, Handle};
+use crate::bindings::{self, Bindings, Definition, Reference};
 use crate::cursor::Cursor;
 
 pub fn execute(file_path_string: &str, version: Version) -> Result<(), CommandError> {
+    // TODO: we probably want the language default path resolver here, not the default one
```

Comment:
I believe we won't have a "default" resolver, and the user will always need to provide one if they want to resolve imports. Otherwise, all imports will stay unresolved.

This command is only used by you for testing. So if it is no longer needed, and the snapshots are enough, maybe we can remove `create()`, `DefaultPathResolver`, etc...

Reply:
Fair point. I only use this command sparingly (once or twice in the last month I think) so I think it can be removed. And along with it, we can remove the default resolver implementation, which is essentially a no-op anyway.

Reply:
I see the command and the resolver are still added in this PR. is that intended?